features/admin.organizations.v1/components/organization-list.tsx

```diff
@@ -325,10 +325,13 @@ export const OrganizationList: FunctionComponent<OrganizationListPropsInterface>
                                     trigger={
                                         (<Icon
                                             data-componentid={ `${ componentId }-org-status-icon` }
-                                            className="mr-2 ml-0 vertical-aligned-baseline"
+                                            className={ "organization-active-icon "
+                                                + (organization.status === "ACTIVE"
+                                                    ? "organization-active-icon-active"
+                                                    : "organization-active-icon-inactive")
+                                            }
```

Comment:
Let's use `classNames` package for this.

Reply:
Fixed with https://github.com/wso2/identity-apps/pull/6640/commits/b34467767ee4bbac6d8f95413ede92091a077118